### ISSUE TITLE
TF: NaN Loss with CNN+RNN Model (multiple inputs, single output)

### ISSUE THREAD

**Issue by brunoalano** [keras-team/keras#11481]
I'm trying to build a multiple-input single-output network using Keras (functional API), but I'm getting NaN loss for every epoch (validation loss too).

```python
input_a = Input(tensor=pageview_input)
input_b = Input(tensor=mouse_input)

# image convolution
im_convl = TimeDistributed(Conv2D(16, (7, 7),
                                  strides=(2, 2),
                                  activation='relu',
                                  padding='same'))(input_b)
im_convl = TimeDistributed(Conv2D(16, (3, 3),
                                  kernel_initializer='he_normal',
                                  activation='relu'))(im_convl)
im_convl = TimeDistributed(MaxPooling2D((2, 2), strides=(2, 2)))(im_convl)

im_convl = TimeDistributed(Conv2D(32, (3, 3),
                                  padding='same',
                                  activation='relu'))(im_convl)
im_convl = TimeDistributed(Conv2D(32, (3, 3),
                                  padding='same',
                                  activation='relu'))(im_convl)
im_convl = TimeDistributed(MaxPooling2D((2, 2), strides=(2, 2)))(im_convl)

im_convl = TimeDistributed(Conv2D(64, (3, 3),
                                  padding='same',
                                  activation='relu'))(im_convl)
im_convl = TimeDistributed(Conv2D(64, (3, 3),
                                  padding='same',
                                  activation='relu'))(im_convl)
im_convl = TimeDistributed(MaxPooling2D((2, 2), strides=(2, 2)))(im_convl)

im_convl = TimeDistributed(Flatten())(im_convl)

# sequence encoding
sequence_rnn = LSTM(32, return_sequences=True)(input_a)
sequence_rnn = Dropout(0.1)(sequence_rnn)

# full model
concat_img_and_pv = concatenate([sequence_rnn, im_convl])
full_model = LSTM(64, return_sequences=False)(concat_img_and_pv)
full_model = Dense(1, activation='sigmoid')(full_model)

# build the model
model = Model(inputs=[input_a, input_b], outputs=full_model)
model.compile(loss='binary_crossentropy', optimizer='rmsprop', metrics=['accuracy'])
```

```python
input_a.shape = (?, timesteps, features_by_each_timestep)
input_b.shape = (?, timesteps, image_height, image_width, 1) # grayscale image
y.shape = (?, 1)
```

I already tried with other optimizers like Adam and Nadam. I also tested using Tensorback, directly in the TF Session, and got the same issue.

Info:

- Python Version: Python 3.6.5
- TF Version: v1.11.0-rc2-4-gc19e29306c 1.11.0
- Keras Version: 2.2.4
- Keras Backend: Tensorflow

**Comment by brunoalano** [comment]
I also tried to change it to a `categorical_crossentropy`, and encode as one_hot of 2 classes (true / false). It also gives NaN as loss.

**Comment by kevinluvian** [comment]
any updates regarding this issue?
